bitrise.yml

```diff
@@ -46,6 +46,6 @@ workflows:
 
             set -e
 
-            ./samples/client/petstore/swift4/swift4_test_all.sh && ./samples/client/test/swift4 && sh swift4_test_all.sh && exit ${PIPESTATUS[0]}
+            ./samples/client/petstore/swift4/swift4_test_all.sh && cd ./samples/client/test/swift4 && /bin/sh swift4_test_all.sh && exit ${PIPESTATUS[0]}
```

Comment:
Sorry, this was a mistake of mine, what do you think instead of `cd` and `sh` do something more direct like this?
`./samples/client/petstore/swift4/swift4_test_all.sh && ./samples/client/test/swift4/swift4_test_all.sh && exit ${PIPESTATUS[0]}`

Reply:
Sure, we can do that.